.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: dart
+sudo: false
+dart:
+  - dev
+  - stable
+  - 1.24.2
+  - 1.23.0
+  - 1.22.1
+
+dart_task:
```

Comment:
It's probably overkill to run these in three separate builders; this package doesn't have that many tests.